src/cdk-experimental/menu/menu-group.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Directive, Output, OnDestroy} from '@angular/core';
+import {Subject} from 'rxjs';
+import {CdkMenuItem} from './menu-item';
+
+/**
+ * Directive which provides grouping logic for CdkMenuItem components marked with role
+ * menuitemradio.
+ * Siblings within the element are part of the same RadioGroup and behave as such.
+ */
+@Directive({
+  selector: '[cdkMenuGroup]',
+  exportAs: 'cdkMenuGroup',
+  host: {
+    // as per aria spec
```

Comment:
I'm not sure we need to comment this, though if we're going to we should link to the spec

Reply:
Do you think its necessary to provide a link?